dev/docker/ballista.dockerfile

```
@@ -49,7 +49,9 @@ ARG PROTOC_VERSION=21.4
 RUN mkdir /tmp/protoc
 WORKDIR /tmp/protoc
 
-RUN export PROTO_ZIP="protoc-${PROTOC_VERSION}-linux-x86_64.zip" && \
+RUN CPU_ARCH=`uname -m` && \
+  if [ $CPU_ARCH=="aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
```

The condition here is always true for me unless I add spaces around the `==`.

```suggestion
  if [ $CPU_ARCH == "aarch64" ]; then CPU_ARCH="aarch_64"; fi && \
```

Here is how I tested this:

test 1

```bash
#!/bin/bash
CPU_ARCH=`uname -m`
echo $CPU_ARCH
if [ $CPU_ARCH=="aarch64" ]; then CPU_ARCH="aarch_64"; fi
echo $CPU_ARCH
```

Produces:

```
x86_64
aarch_64
```

test 2

```bash
#!/bin/bash
CPU_ARCH=`uname -m`
echo $CPU_ARCH
if [ $CPU_ARCH == "aarch64" ]; then CPU_ARCH="aarch_64"; fi
echo $CPU_ARCH
```

produces

```
x86_64
x86_64
```

---

and I see the opposite behavior on Mac :confused: